src/schemas/validation/schema.yaml

```diff
@@ -68,6 +71,7 @@ $defs:
       description:
         type: string
       termsOfService:
+        $comment: resolved against the resolved version of $self
```

Comment:
FWIW, I had to think about whether Info/License/Contact should be OAD URI-based or API URL-based.  Taking a look at these fields, we do say "URI", and that's what I was expecting to find.  I wonder if we should make "URI" a link to 4.1.2.2 Relative References in API Description URIs for added clarity?